.env.example

```
@@ -12,3 +12,7 @@ DATABASE_URL=sqlite:///db.sqlite3
 # Email Configuration for OTP (Optional)
 EMAIL_HOST_USER=
 EMAIL_HOST_PASSWORD=
+
+# Email Backend (Optional, defaults to console for local dev)
+# For production, set to: django.core.mail.backends.smtp.EmailBackend
+EMAIL_BACKEND=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)